scheduler/cmd/scheduler/main.go

```diff
@@ -23,6 +23,7 @@ import (
 	_ "k8s.io/component-base/metrics/prometheus/clientgo" // for rest client metric registration
 	_ "k8s.io/component-base/metrics/prometheus/version"  // for version metric registration
 	"k8s.io/kubernetes/cmd/kube-scheduler/app"
+	wasm "sigs.k8s.io/kube-scheduler-wasm-extension/scheduler/plugin"
```

Comment:
this broke earlier. After this PR is in I will setup CI to test all the src directories.